os/pm/pm_dvfs.c

```diff
@@ -80,7 +80,7 @@
  *
  * Description:
  *   This internal function is called to reduce the frequency of operation
- *   of the secondary AP core of AI_Dual chipset.
```

Comment:
This internal function is called to tune (ie. Not recommend to use reduce here because it is dynamic tuning) the clock frequency of the application core. Do note that, it will only affect the power consumption during active mode, it can be applied on some cases which low loading tasks is expected. Please keep in mind that there is a tradeoff between processing speed and cpu active time interval. Which means, we will expect to see lower power consumption during that period of time (ie. reduced clock frequency), but the time interval to handle some task will also consume longer time.